util/ci_utils.sh

```diff
@@ -208,6 +224,13 @@ build_pip_conda_package() {
     else
         echo "Building for a new Open3D release"
     fi
+    if [[ "build_azure_kinect" =~ ^($options)$ ]]; then
+        echo "Azure Kinect enabled in Python wheel."
+        BUILD_AZURE_KINECT=ON
+    else
+        echo "Azure Kinect disabled in Python wheel."
+        BUILD_AZURE_KINECT=OFF
+    fi
```

Comment:
I don't think we need this as an option. Let's always have this enabled. I'll do the same for RealSense.